libexec/alluxio-config.sh

```diff
@@ -120,3 +128,4 @@ ALLUXIO_WORKER_JAVA_OPTS+=" -Dalluxio.logger.type=${ALLUXIO_WORKER_LOGGER:-WORKE
 # Client specific parameters based on ALLUXIO_JAVA_OPTS.
 ALLUXIO_USER_JAVA_OPTS+=${ALLUXIO_JAVA_OPTS}
 ALLUXIO_USER_JAVA_OPTS+=" -Dalluxio.logger.type=USER_LOGGER"
+ALLUXIO_USER_JAVA_OPTS+=" -Dalluxio.master.audit.logger.type=Null"
```

Comment:
Is it possible to set `alluxio.master.audit.logger.type=Null` by default in the `log4j.properties` file? https://stackoverflow.com/a/29514364 suggests that this could work